safety-quiz/templates/login.html

```diff
@@ -1,26 +1,58 @@
 {% extends 'layout.html' %}
 {% block page %}
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
I believe this import is unnecessary. Could you try removing this and see if it still works? JQuery is supposed to be loaded for all templates in safety-quiz

Reply:
Yes, It's working without the import
I'll update this